integrations/sbt-bloop/src/main/scala/bloop/integrations/sbt/SbtBloop.scala

```diff
@@ -784,7 +788,10 @@ object BloopDefaults {
       }
     } else {
       Def.task {
-        val config = Config.JvmConfig(Some(javaHome.toPath), javaOptions.toList)
+        val isForkedExecution = (Keys.fork in configuration in forkScopedTask).value
+        val workingDir = if (isForkedExecution) Keys.baseDirectory.value else rootBaseDirectory
+        val extraJavaOptions = List(s"-Duser.dir=${workingDir.getAbsolutePath}")
+        val config = Config.JvmConfig(Some(javaHome.toPath), (javaOptions ++ extraJavaOptions).toList)
```

Comment:
You might want to swap the order so users can still override `-Duser.dir`
```suggestion
        val config = Config.JvmConfig(Some(javaHome.toPath), (extraJavaOptions ++ javaOptions ++).toList)
```